src/components/Navigation.js

```diff
@@ -54,6 +54,9 @@ class Navigation extends React.Component {
               <li className="nav-item">
                 <NavLink to="/custom_tokens/" exact className="nav-link" activeClassName="active" activeStyle={{ fontWeight: 'bold' }}>{t`Custom tokens`}</NavLink>
               </li>
+              <li className="nav-item">
+                <NavLink to="/addresses/" exact className="nav-link" activeClassName="active" activeStyle={{ fontWeight: 'bold' }}>{t`Addresses`}</NavLink>
```

Comment:
If we go with number 3, we should remove this change.

Reply:
Removed